ethers-contract/ethers-contract-abigen/src/multi.rs

```diff
@@ -13,7 +13,11 @@ use std::{
 use toml::Value;
 
 /// The default ethers dependency to generate.
-const DEFAULT_ETHERS_DEP: &str = "ethers = { git = \"https://github.com/gakonst/ethers-rs\", default-features = false, features = [\"abigen\"] }";
+const DEFAULT_ETHERS_DEP: &str = concat!(
+    "ethers = { version = \"",
+    env!("CARGO_PKG_VERSION"),
+    "\", default-features = false, features = [\"abigen\"] }"
+);
```

Comment:
this will cause issues with foundry because foundry uses ethers master to generate bindings that then depend on crates.

so until we don't use ethers from crates in foundry, I recommend against this.

Reply:
makes sense, closing until then